### PR TITLE
prJobTest

### PR DESCRIPTION
**Pull request by laGenteEstaMuyLoca81**

### CODE REVIEW

**Comment by bickralf** [comment]
retest this please

**Comment by bickralf** [comment]
retest this please

**Comment by bickralf** [comment]
retest this please

**Comment by bickralf** [comment]
retest this please

File: src/test/java/de/devbliss/apitester/factory/impl/EntityBuilderTest.java [hunk 1]
```diff
@@ -52,7 +52,7 @@ public void setName(String name) {
 
     @Test
     public void buildEntityFromStringPayload() throws Exception {
-        final String payload = "spitting cobra";
```
Comment (jopek) [inline src/test/java/de/devbliss/apitester/factory/impl/EntityBuilderTest.java]: lalala - kommentar bitte ignorieren, gpullr kommentar event test

Comment (jopek) [inline src/test/java/de/devbliss/apitester/factory/impl/EntityBuilderTest.java]: another one

**Comment by jopek** [comment]
issue comment!!!11111!!!elf

**Pull request closed without merging**